src/shilpa/Assignment_7/Shop.java

```diff
@@ -0,0 +1,174 @@
+//Assignment  7  : 30th March 2021 
+
+//Create a system (Shop) which maintains each item's quantity.
+
+package shilpa.Assignment_7;
+
+public class Shop {
+	int maggieQuantity, dosaQuantity, oilQuantity, panipuriQuantity, masalaQuantity;
+
+	void Inventory(int maggie,int dosa, int oil,int panipuri,int masala ) {
+		maggieQuantity = maggie;
+		dosaQuantity = dosa;
+		oilQuantity = oil;
+		panipuriQuantity = panipuri;
+		masalaQuantity = masala;	
+	}
+
+	void customer(int maggieOrder,int dosaOrder, int oilOrder,int panipuriOrder,int masalaOrder) {
+		if(maggieQuantity == 0) 
```

Comment:
If person pass 0 packs of maggie then it doesn't mean we show message as running out of stock right?